cantabular/dimensions.go

```diff
@@ -224,3 +227,67 @@ func (c *Client) GetParents(ctx context.Context, req GetParentsRequest) (*GetPar
 
 	return &resp.Data, nil
 }
+
+// GetGeographyDimensionsInBatches performs a graphQL query to obtain all the geography dimensions for the provided cantabular dataset.
+// The whole response is loaded to memory.
+func (c *Client) GetGeographyDimensionsInBatches(ctx context.Context, datasetID string, batchSize, maxWorkers int) (dataset *gql.Dataset, err error) {
```

Comment:
Please don't use named return values unless necessary, especially don't mix named return values in addition to returning explicit values